cartridges/int_klarna_payments_sfra/cartridge/client/default/js/klarnaOsm.js

```diff
@@ -4,7 +4,7 @@ $(function () {
     $('body').on('product:afterAttributeSelect', function (e, response) {
         if (response.data.product.price.sales) {
             document.getElementsByTagName('klarna-placement')[0].setAttribute('data-purchase-amount', Math.round(response.data.product.price.sales.value * 100));
-            window.KlarnaOnsiteService.push({ eventName: 'refresh-placements' });
+            window.Klarna.OnsiteMessaging.refresh();
```

Comment:
Please add this change to the changelog since I can't find it there.

Reply:
issue fixed (I've added 'Replace deprecated window.KlarnaOnsiteService.push with window.Klarna.OnsiteMessaging.refresh.' to the changelog).